### PR TITLE
Add ifeval to remove broken link

### LINKED ISSUE #256: Broken xref in Managing Hosts guide
https://docs.theforeman.org/master/Managing_Hosts/index-foreman.html#sect-Red_Hat_Satellite-Managing_Hosts-Adding_a_Host_to_a_Host_Collection 

I will try fix it this week

### PR DESCRIPTION
**Pull request by melcorr**
* Add note about auditing of host collection

Fixes #256 

See: 

https://community.theforeman.org/t/missing-audit-record-for-host-collection-changes/20390/3?u=mcorr

### CODE REVIEW

File: guides/doc-Managing_Hosts/topics/Adding_Systems_to_System_Groups.adoc [hunk 1]
```diff
@@ -6,7 +6,15 @@ The following procedure shows how to add hosts to host collections.
...
-A host must be registered to {ProjectName} in order to add it to a Host Collection. For more information about registering hosts, xref:Registering_Hosts[].
+ifeval::["{build}" == "satellite"]
+A host must be registered to {ProjectName} to add it to a Host Collection. For more information about registering hosts, xref:Registering_Hosts[].
+endif::[]
+
+ifeval::["{build}" != "satellite"]
+A host must be registered to {ProjectName} to add it to a Host Collection.
+endif::[]
+
+Mote that if you add a host to a host collection, the {Project} auditing system does not log the change.
```
Comment (spetrosi) [inline guides/doc-Managing_Hosts/topics/Adding_Systems_to_System_Groups.adoc:17]:
```suggestion
Note that if you add a host to a host collection, the {Project} auditing system does not log the change.
```
:-)

Comment (spetrosi) [inline guides/doc-Managing_Hosts/topics/Adding_Systems_to_System_Groups.adoc:17]:
```suggestion
Note that if you add a host to a host collection, the {Project} auditing system does not log the change.
```

File: guides/doc-Managing_Hosts/topics/Removing_Systems_from_System_Groups.adoc [hunk 2]
```diff
@@ -3,6 +3,8 @@
 
 The following procedure shows how to remove hosts from host collections.
 
+Mote that if you remove a host from a host collection, the host collection record in the database is not modified so the {Project} auditing system does not log the change.
```
Comment (spetrosi) [inline guides/doc-Managing_Hosts/topics/Removing_Systems_from_System_Groups.adoc:6]:
```suggestion
Note that if you remove a host from a host collection, the host collection record in the database is not modified so the {Project} auditing system does not log the change.
```

**Review by spetrosi — CHANGES_REQUESTED** [review]

**Comment by melcorr** [comment]
Thanks, Sergei!
That was shocking!!

**Comment by spetrosi** [comment]
I assume this also applies to 6.7 and 6.8 versions, @melcorr?

**Review by spetrosi — APPROVED** [review]

**Comment by melcorr** [comment]
It does, @spetrosi ! It applies to all. It just caused some confusion to a user so I thought why not add a note.

**Pull request merged**